src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  color: #000;
+
+
+  --color-blue: #00acdc;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.product-photo {
```

Comment:
'product-photo' is a BEM-block, so don't use margins for it. You can give this element a separate class as a BEM-element, and use it for margins, e.g. card__product-photo